### PR TITLE
The country / region of mail sent from the admin panel is translated into the admin locale.

### LINKED ISSUE #26521: The country / region of mail sent from the admin panel is translated into the admin locale.
### Preconditions (*)

1.  Magento2.3.3
2. PHP7.2
3.  Two stores (JP & EN)
4.  Admin user locale is ja_JP
### Steps to reproduce (*)

1. place order  in EN store.
2. send order Email from admin panel using admin user locale ja_JP.

### Expected result (*)
Displayed in English.

スクリーンショット 2020-01-24 17 48 39

### Actual result (*)
Country is Displayed in Japanese (admin user locale) even though this order is placed on EN store.

スクリーンショット 2020-01-24 17 48 32 

if I change admin panel locale to en_US from ja_JP , it will work fine.

When sending mail from admin panel, Magento\Backend\Model\Locale\Resolver::setLocale method is used. And it force to set admin user local to county & region .

### PR DESCRIPTION
**Pull request by engcom-Charlie**
### Fixed Issues (if relevant)

1. Fixes magento/magento2#26521
### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by engcom-Charlie** [comment]
@magento run all tests

File: app/code/Magento/Customer/Block/Address/Renderer/DefaultRenderer.php [hunk 1]
```diff
@@ -172,9 +172,9 @@ public function renderArray($addressAttributes, $format = null)
             }
             $attributeCode = $attributeMetadata->getAttributeCode();
             if ($attributeCode == 'country_id' && isset($addressAttributes['country_id'])) {
-                $data['country'] = $this->_countryFactory->create()->loadByCode(
-                    $addressAttributes['country_id']
-                )->getName();
+                $data['country'] = $this->_countryFactory->create()
+                    ->loadByCode($addressAttributes['country_id'])
+                    ->getName(isset($addressAttributes['locale']) ? $addressAttributes['locale'] : null);
```
Comment (eduard13) [inline app/code/Magento/Customer/Block/Address/Renderer/DefaultRenderer.php]: Lets use the null coalescing operator
```suggestion
                    ->getName($addressAttributes['locale'] ?? null);
```

**Review by eduard13 — CHANGES_REQUESTED** [review]
Thank you @engcom-Charlie for the fix, please review the below comment and please note that the following Unit Test has also to be adjusted
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Sales/Test/Unit/Model/Order/Address/RendererTest.php

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Review by eduard13 — APPROVED** [review]
Good job 👍

**Comment by engcom-Bravo** [comment]
:heavy_check_mark: QA Passed

### Manual testing scenario
1. Create two new Stores and Store views, name them _English_ and _Japanese_
stores
2. Go to _Stores - Configuration - General - Locale Options_
3. Set English United States **Locale** for the English store view **Scope**, and Japan **Locale** for the Japanese Store view **Scope**
4. Go to **Admin Account Settings** and change the **Interface Locale** to Japan
5. From the Storefront, open English store
6. Place an order specifying the customer address country, United States
7. From Admin Panel, go to _Sales - Orders_ and open the previously created order
8. Click **Send Email**

**BEFORE** applying changes provided in the PR, the country was displayed in Japanese symbols
country_translate

**AFTER** switching to the PR, the country is displayed in English language
country_translate_after

**Pull request merged**